crates/block-sys/src/lib.rs

```diff
@@ -24,11 +24,9 @@
 #![cfg_attr(feature = "unstable-docsrs", feature(doc_auto_cfg, doc_cfg_hide))]
 #![cfg_attr(feature = "unstable-docsrs", doc(cfg_hide(doc)))]
 
+#[cfg(feature = "std")]
 extern crate std;
 
-#[cfg(not(feature = "std"))]
-compile_error!("The `std` feature currently must be enabled.");
-
```

Comment:
Please do roughly the same setup as I did in [this PR to the `url` project](https://github.com/servo/rust-url/pull/722/files#diff-8d002470387f8d121e1b1c8883fe8c7c79b8309ad368499e03af0a6d1358abb5) (for both `alloc` and `std`, since neither are required).